app/controllers/defendants_controller.rb

```diff
@@ -47,6 +56,12 @@ def prosecution_case_reference
 
   private
 
+  def handle_link_failure(message, exception = nil)
+    logger.warn "LINK DEFENDANT FAILURE (params: #{@unlink_attempt.as_json}): #{message}"
+    @unlink_attempt.errors.add(:maat_reference,
```

Comment:
I'm not sure `maat_reference` is the right field to add this error to here, since there's no form field for it. I'd go either with `:base` or maybe `:reason_code`?

Reply:
yep, :base sounds good to me!